.idea/libraries/junit_junit_4_11.xml

```diff
@@ -0,0 +1,11 @@
+<component name="libraryTable">
```

Comment:
Let's add .gitignore and ignore .idea directory and amend this commit.

Reply:
out directory as well.

Reply:
okay